src/plugin/code.js

```diff
@@ -205,6 +280,25 @@ figma.on("selectionchange", () => {
       type: "RENDER_DIFFERENCE_INFORMATION",
       content: "UNCHANGED_NODE",
     });
+
+    if (totalFrameCount) {
+      figma.ui.postMessage({
+        type: "FRAME_PAGINATION",
+        content: {
+          result: true,
+          currentCount: 0,
+          frameCounts: totalFrameCount,
+        },
+      });
+    } else {
+      figma.ui.postMessage({
+        type: "FRAME_PAGINATION",
+        content: {
+          result: false,
+          frameCounts: "- / -",
+        },
+      });
+    }
```

Comment:
- `P2`
early return 패턴을 적용해보면 좋을 것 같습니다!
```suggestion
    if (totalFrameCount) {
      figma.ui.postMessage({
        type: "FRAME_PAGINATION",
        content: {
          result: true,
          currentCount: 0,
          frameCounts: totalFrameCount,
        },
      });
      
      return;
    }
    
    figma.ui.postMessage({
      type: "FRAME_PAGINATION",
      content: {
        result: false,
        frameCounts: "- / -",
      },
    });
```

Reply:
early return 을 적용하니 depth를 줄이고 가독성이 향상되었네요 👍